content/blog/sprint-review-2217.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Wazo Platform 22.17 Released
+date: 2022-12-14T21:33:00
+author: The Wazo Authors
+category: Wazo Platform
+tags: wazo-platform, development
+slug: release-review-2217
+status: published
+---
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 22.17 release.
+
+## Technical Features
+- **Python3**: Migrated wazo-agid and wazo-confgend services to Python 3. They were the latest services written in Python 2
+- **Asterisk**: Asterisk has been upgraded to 19.7.1. See the [Asterisk release announcement](https://www.asterisk.org/asterisk-news/asterisk-19-7-1-now-available-security-release/)
+
+## Ongoing Features
+- **New user API**: We are working on a new high level API endpoint to create a user and all of its related resources (user, line, extension, voicemail, incoming call) in a single HTTP request.
+- **Major upgrade**: We are preparing the upgrade to Debian 11 Bullseye. This requires migrating the remaining code written in Python 2 to Python 3.
```

Comment:
We wrote in a previous bullet point that the last services were ported to Python 3. Would that make sense to remove the sentence?